vnext/Microsoft.ReactNative.SharedManaged/JSValueReaderGenerator.cs

```diff
@@ -19,6 +19,8 @@ public static class JSValueReaderGenerator
     private static readonly Assembly s_currentAssembly = typeof(JSValueReaderGenerator).Assembly;
 
     private static readonly ConcurrentDictionary<Assembly, bool> m_registerdAssemblies = new ConcurrentDictionary<Assembly, bool>();
```

Comment:
>m_ [](start = 65, length = 2)

s_ ?

Reply:
Auto behavior of the coding conventions I used the last 12 years.. `s_` is for static members `m_` for instance members.

Reply:
Seems the convention mostly in the codebase already. Seems m_registerdAssemblies  is the 1-off.. I'll update that one.